lib/msal-browser/src/error/BrowserAuthError.ts

```diff
@@ -184,6 +184,14 @@ export const BrowserAuthErrorMessage = {
     nativePromptNotSupported: {
         code: "native_prompt_not_supported",
         desc: "The provided prompt is not supported by the native platform. This request should be routed to the web based flow."
+    },
+    chromiumStorageException: {
+        code: 22,
```

Comment:
Do we have to define this ourselves or is there some other variable we can pull from? Are we confident this is stable?